### ISSUE TITLE
CCDA Service compile is broken node version > v18.x

### ISSUE THREAD

**Issue by sjpadgett** [openemr/openemr#7528]
### Describe the bug
Node server app serveccda.js fails to start
### To Reproduce
npm install with version greater than 18.x or current recommended 20.10.0 won't work because evidentially I didn't update the lock file for new versions.
I'm testing now if the compiled version on 20.10.0 is backwards compatible with lower versions or minor releases in v20.x.0 of node.

Works on v18.20.1 So use this node version until I get the solution into the patch pipeline. @juggernautsei please note. Or delete package lock file and compile will work.

RE: I'm sorry if reading this before this edit but in order to use serccda.js before a new patch you must:
- delete the following directories: `ccdaservice/oe-blue-button-generate/node_modules`, `ccdaservice/node_modules`
- then delete `package-lock.json`
- cd `oe-blue-button-generate and npm install` // not sure this is needed any longer and will investigate.
- cd ..
- then `npm install`

That should do it. again @juggernautsei work around for your current use case.

I will use node v20.11.1 for my next package release of CCDA because this is what docker currently uses but say v20.15.0 will also work on 20.11.1 compiled version etc.

### Steps to reproduce the behavior:

compile with node version 20.x

### Client configuration

- Browser:
- OpenEMR version:
- Operating system: all three we support

**Comment by sjpadgett** [comment]
This is interesting! It seems like the libxmljs dependency needed is installed in two places however when I split off from BB project I left dependency in ccdaservice/oe-blue-button-generate/node_modules causing version difference in root project directory to be different. Anyway nobody cares why and I waste my time explaining it.
Suffice to say looks like deleting ccdaservice/oe-blue-button-generate/node_modules will fix version compiled updates to say 20.10.1

**Comment by sjpadgett** [comment]
Turns out nothing really broken but my local repo and ego where I carried around ccdaservice/oe-blue-button-generate/node_modules which broke my testing. 
Current in the world is okay on most all versions. I though easy dev compiled ccda but no.

Wasn't complete waste of time as updated all dependencies and fixed a long standing moment ISO date error that was quite annoying because I knew it was there for past 5 years!
Just one lousy date from CCM!
